recipes/dict2css/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dict2css" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dict2css-{{ version }}.tar.gz
+  sha256: 1e8b1bf580dca2083198f88a60ec88c878a8829d760dfe45483ef80fe2905117
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
```

Comment:
@domdfcoding why do we need pip here?

Reply:
`grayskull` included it automatically when generating the recipe. I think it's needed because `pip install . -vv` is used to build the package.

Reply:
It's not needed for runtime though.

Reply:
Absolutely :), just double checking!